lib/libBixolonUPOS/BixolonUPOS.h

```diff
@@ -0,0 +1,15 @@
+//
```

Comment:
What's the purpose of this file?

Reply:
It's the file in printer sdk from Bixolon.
Can't be sure of the purpose.

Reply:
That's not an acceptable answer. We should not add files in the project if we don't know the purpose. Did you try to remove it?

Reply:
@EricGlaenzer something to be said for leaving it if it is part of the Bixilon SDK though, I would think.

Reply:
@EricGlaenzer 
This file is part of the Bixilon SDK.
Can't be sure removing this file doesn't make any problem.
I think it's better don't make any change on Bixilon SDK files.

Reply:
tried to compile and run after delete this file.
and couldn't find error.
removed this file.